llm/server.go

```diff
@@ -221,7 +221,7 @@ func NewLlamaServer(gpus gpu.GpuInfoList, model string, ggml *GGML, adapters, pr
 		params = append(params, "--memory-f32")
 	}
 
-	flashAttnEnabled := envconfig.FlashAttention
+	flashAttnEnabled := envconfig.FlashAttention()
```

Comment:
How do I know when to use the var `FlashAttention` vs the function `FlashAttention()`?

Reply:
`envconfig.FlashAttention` is a function so it's unclear what var you're referring to

Reply:
Misread the variable declaration, all good